web-client/src/presenter/computeds/formattedTrialSessionDetailsForFilteredEligibleCases.test.js

```diff
@@ -0,0 +1,145 @@
+import { formattedTrialSessionDetailsForFilteredEligibleCases } from './formattedTrialSessionDetailsForFilteredEligibleCases';
+import { runCompute } from 'cerebral/test';
+
+jest.mock('./formattedTrialSessionDetails', () => {
+  return {
+    __esModule: true,
+    formattedTrialSessionDetails: () => ({
+      formattedEligibleCases: [
+        {
+          caseCaption: 'testPetitioner1, Petitioner',
+          caseTitle: 'testPetitioner1',
+          caseType: 'CDP (Lien/Levy)',
+          docketNumber: '103-20',
+          docketNumberSuffix: 'L',
+          docketNumberWithSuffix: '103-20L',
+          entityName: 'EligibleCase',
+          inConsolidatedGroup: false,
+          irsPractitioners: [],
+          isDocketSuffixHighPriority: true,
+          leadCase: false,
+          privatePractitioners: [],
+          qcCompleteForTrial: {},
+        },
+        {
+          caseCaption: 'testPetitioner2, Petitioner',
+          caseTitle: 'testPetitioner2',
+          caseType: 'Worker Classification',
+          docketNumber: '108-19',
+          docketNumberSuffix: null,
+          docketNumberWithSuffix: '108-19',
+          entityName: 'EligibleCase',
+          inConsolidatedGroup: false,
+          irsPractitioners: [],
+          isDocketSuffixHighPriority: false,
+          leadCase: false,
+          privatePractitioners: [],
+          qcCompleteForTrial: {},
+        },
+        {
+          caseCaption: 'testPetitioner3, Petitioner',
+          caseTitle: 'testPetitioner3',
+          caseType: 'Deficiency',
+          docketNumber: '101-20',
+          docketNumberSuffix: 'S',
+          docketNumberWithSuffix: '101-20S',
+          entityName: 'EligibleCase',
+          inConsolidatedGroup: false,
+          irsPractitioners: [],
+          isDocketSuffixHighPriority: true,
+          leadCase: false,
+          privatePractitioners: [],
+          qcCompleteForTrial: {},
+        },
+        {
+          caseCaption: 'testPetitioner4, Petitioner',
+          caseTitle: 'testPetitioner4',
+          caseType: 'CDP (Lien/Levy)',
+          docketNumber: '110-20',
+          docketNumberSuffix: 'SL',
+          docketNumberWithSuffix: '110-20SL',
+          entityName: 'EligibleCase',
+          inConsolidatedGroup: false,
+          irsPractitioners: [],
+          isDocketSuffixHighPriority: true,
+          leadCase: false,
+          privatePractitioners: [],
+          qcCompleteForTrial: {},
+        },
+      ],
+    }),
+  };
+});
+
+describe('formattedTrialSessionDetailsForFilteredEligibleCases', () => {
+  it('should display all cases when filter is falsy', () => {
+    const result = runCompute(
+      formattedTrialSessionDetailsForFilteredEligibleCases,
+      {
+        state: {
+          screenMetadata: {
+            eligibleCasesFilter: {
+              hybridSessionFilter: undefined,
+            },
+          },
+        },
+      },
+    );
+
+    expect(result.formattedEligibleCases).toHaveLength(4);
```

Comment:
⛏️ Could the expected value be the length of the mock data being returned, in case it's ever changed?

Reply:
done...thanks